desktop/cmp/input/Select.ts

```diff
@@ -52,6 +52,13 @@ export interface SelectProps extends HoistProps, HoistInputProps, LayoutProps {
     /** True (default) to close the menu after each selection. */
     closeMenuOnSelect?: boolean;
 
+    /**
```

Comment:
This docstring sounds a little too much like the spec ---

`Value to use when the input is empty. Defaults to null.
Typical usage is to set to [] for multi-selects.`